packages/ui/src/primitives/card.tsx

```diff
@@ -132,7 +132,7 @@ export const FooterAction = React.forwardRef<HTMLDivElement, React.HTMLAttribute
       <div
         ref={forwardedRef}
         {...props}
-        className={cn('px-6 py-4', className)}
+        className={cn('px-6 py-4 border-b border-gray-a6 ', className)}
```

Comment:
Moves the border to the conditionally rendered element for when the FooterAction isn't used 
<img width="586" alt="Screenshot 2024-06-11 at 4 12 47 PM" src="https://github.com/clerk/javascript/assets/825855/71152410-9f35-4921-a35f-51705c5398fb">